tests/test_blueice_extended_model.py

```diff
@@ -70,6 +70,11 @@ def get_expectation_values(self):
             expectation_values.append(this_expectation_dict)
         return expectation_values
 
+    def test_get_all_source_names(self):
+        """Test of the get_all_source_names method."""
+        for model in self.models:
+            model.get_all_source_names()
+
```

Comment:
Not sure if checking whether it doesn't crash is a sensible test 😅 Maybe load the config and check manually or something like that? Otherwise this is a bit pointless or not?

Reply:
Added the consistency check in pytest.